application/libraries/FormExtension/Inputs/GlobalSettingsRenderer.php

```diff
@@ -5,11 +5,35 @@
 class GlobalSettingsRenderer extends DefaultBaseRenderer
 {
     /**
-     * @param RawHtmlInput|BaseInput $input
+     * @param RawHtmlInput|BaseInput|FileInput $input
      */
     public function run($input): string
     {
         switch (true) {
+            case $input instanceof FileInput:
+                $id = $input->getId() ? 'id="' . $input->getId() . '"' : '';
+                $accept = !empty($input->getAccept())
```

Comment:
$input->getAcceptAsHtml()?

Reply:
Or bake :joy: But that's apparently in this class, not the input.

Reply:
Your wish is my command.